README.md

```diff
@@ -1,26 +1,20 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Jenkins-k8s Operator
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing [Jenkins](https://www.jenkins.io/) Agent on machines and configurable to use a Jenkins charm deployed in another Juju model.
```

Comment:
s/and/, ?

Reply:
good point, I updated the wording on this